nft_ingester/src/program_transformers/mod.rs

```diff
@@ -63,8 +63,10 @@ impl ProgramTransformer {
         order_instructions(ref_set, tx)
     }
 
+    #[allow(clippy::borrowed_box)]
     pub fn match_program(&self, key: &FBPubkey) -> Option<&Box<dyn ProgramParser>> {
-        self.matchers.get(&Pubkey::new(key.0.as_slice()))
+        self.matchers
+            .get(&Pubkey::try_from(key.0.as_slice()).expect("valid key from FlatBuffer"))
```

Comment:
Even though this should probably never fail should we instead log a failure and just return `None` so that the process doesn't get killed?

Reply:
I guess we can leave it as-is although I would prefer for it to return an error instead of panic.  But it is not a blocker as its current behavior today.

Reply:
fixed